napari/layers/_shapes_layer/shape_list.py

```diff
@@ -657,3 +657,36 @@ def to_labels(self, labels_shape=None, shape_type=None):
                     labels[mask] = index[ind]
 
         return labels
+
+    def to_xml(self, shape_type=None):
+        """Returns a list of xml elements defining each shape according to the
+        svg specification. Z ordering of the shapes will be taken into account.
+        Passing a `shape_type` argument leads to only xml from that particular
```

Comment:
This does not belong in the summary. Please move it to the parameter descriptions.

Reply:
removed